src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/_ManageEvidenceNotesHeaderAndTabs.cshtml

```diff
@@ -34,6 +34,10 @@
 }
 
 <ul class="govuk-tabs__list">
+    @this.WeeeGds().Tab("Summary",
```

Comment:
For future proofing probably stick 

if (!@Model.SchemeInfo.IsBalancingScheme) around this tab as the PBS one is slightly different